python/cugraph/tests/test_maximum_spanning_tree.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+
+import pytest
+
+import cugraph
+from cugraph.tests import utils
+from cugraph.utilities import check_nx_graph
+from cugraph.utilities import cugraph_to_nx
+import rmm
+import cudf
+import time
+import numpy as np
+import pandas as pd
+
+# Temporarily suppress warnings till networkX fixes deprecation warnings
+# (Using or importing the ABCs from 'collections' instead of from
+# 'collections.abc' is deprecated, and in 3.8 it will stop working) for
+# python 3.7.  Also, this import networkx needs to be relocated in the
+# third-party group once this gets fixed.
+import warnings
+
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    import networkx as nx
+
+print("Networkx version : {} ".format(nx.__version__))
+
+
+def compare_mst(mst_cugraph, mst_nx):
```

Comment:
Duplicated with `test_minimum_spanning_tree.py`. Ideally it should be moved to a utils file shared by the two, or one file can host the tests and the other file can import

Reply:
Moved to test/utils.py 0c6b9eba